crates/lib3h/src/transport/transport_multiplex/mplex.rs

```diff
@@ -326,10 +326,20 @@ impl<
                 }),
             )?;
         } else {
-            self.inner_gateway.as_mut().publish(
+            let orig_req = data.clone();
+            self.inner_gateway.as_mut().request(
```

Comment:
I don't get this.  The `data` is not a request, it's an event, how can you publish it here?

Reply:
layers above (erm below?) were triggering respond with error conditions, leading to un-trackable panics. At least if we have an `error!(` log here, we can diagnose problems.